src/Microsoft.ML.AutoML/AutoMLExperiment/AutoMLExperiment.cs

```diff
@@ -81,6 +108,12 @@ public AutoMLExperiment SetIsMaximize(bool isMaximize)
             return this;
         }
 
+        public AutoMLExperiment SetMaximumMemoryUsageInMegaByte(double value = double.MaxValue)
```

Comment:
>value [](http://example.com/codeflow?start=71&length=5)

what happen if someone passed `NaN` value?

Reply:
I will add a check here to make sure it's either not `NaN` or non-positive.